mobile_app_connector/models/compassion_login.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2018 Compassion CH (http://www.compassion.ch)
+#    @author: Quentin Gigon <gigon.quentin@gmail.com>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+import logging
+
+from odoo import models, api
+from ..mappings.compassion_login_mapping import MobileLoginMapping
+
+logger = logging.getLogger(__name__)
+
+
+class CompassionLogin(models.Model):
+    _inherit = 'res.users'
+
+    @api.model
+    def mobile_login(self, view, username, password, **other_params):
```

Comment:
You can handle the parameters as it was done by Nicolas (see latest version of devel). He added a method to validate required parameters as well.